### PR TITLE
Feature/1123 core vars

### PR DESCRIPTION
**Pull request by xak**
Closes sap/fundamental#1123

Implements CSS vars with fallbacks for core and forms modules

#### Test

There should be no discernable visual changes.

* http://localhost:3030/pages/form-elements
* http://localhost:3030/toggle
* http://localhost:3030/input
* http://localhost:3030/input-group
* http://localhost:3030/checkbox
* http://localhost:3030/calendar
* http://localhost:3030/radio
* http://localhost:3030/toggle

#### Changelog

**New**

* N/A

**Changed**

* To enable the fallbacks, the flag `$fd-support-css-var-fallback` must be set to true in a custom SASS build.

**Removed**

* N/A

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit 0f2e61f8c8dbba48888394ff92d8e1e170c7d87f

https://deploy-preview-1139--fundamental.netlify.com

**Comment by xak** [comment]
Note that the `toggle` cannot be fully migrated since there are some hardcoded colors in the SASS. These need to be addressed and converted to colors in the palette.

File: scss/components/toggle.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// 
+//
```
Comment (ckoutsiaris) [inline scss/components/toggle.scss]: this can be removed?

File: scss/components/toggle.scss [hunk 2]
```diff
@@ -89,7 +88,8 @@ $block: #{$fd-namespace}-toggle;
         left: 0;
         right: 0;
         bottom: 0;
-        background-color: $fd-toggle-background-color;
+        //background-color: $fd-toggle-background-color;
```
Comment (ckoutsiaris) [inline scss/components/toggle.scss]: maybe this can be removed?

**Pull request merged**